imagediffer.js

```diff
@@ -17,14 +27,18 @@ ImageDiffer.prototype.doDiff = function(boxes) {
   return resemble(a)
     .compareTo(b)
     .ignoreNothing()
-    .skip(boxes)
+    .skip(this.skipBoxes)
```

Comment:
Not sure if order matters here, but if it does, then these feel like they are in the wrong order based on the names `includeOnlyBoxes` and `censorBoxes`. I would expect the `censorBoxes` to censor stuff in the `includeOnlyBoxes`.